CMakePresets.json

```diff
@@ -6,6 +6,42 @@
     "patch": 0
   },
   "configurePresets": [
+    {
+      "name": "ci",
+      "displayName": "ci",
+      "description": "Options options for CI build (PR and Main)",
+      "binaryDir": "${sourceDir}/build",
+      "generator": "Ninja",
+      "cacheVariables": {
+        "CMAKE_BUILD_TYPE": "Release",
```

Comment:
We could use `RelWithDebInfo` instead. This would help with debugging when someone installs the package from the dev label.

Reply:
This is not a package build, but we should consider your suggestion for the (to be added) `package` preset.

Reply:
Right